base/tests/test_forms.py

```diff
@@ -371,8 +372,6 @@ async def test_submit_18_minor_to_healthconnect(self):
             "exposure": "not_sure",
             "tracing": True,
             "risk": "moderate",
-            "location": "",
```

Comment:
I haven't looked at all the tests but make sure we test with another age group too

Reply:
Yeah